wp-parsely-2.5/src/class-parsely.php

```diff
@@ -855,11 +946,32 @@ public static function post_has_trackable_status( $post ) {
 		 * @param string[]    $trackable_statuses The list of post statuses that are allowed to be tracked.
 		 * @param int|WP_Post $post               Which post object or ID is being checked.
 		 */
-		$statuses = apply_filters( 'wp_parsely_trackable_statuses', array( 'publish' ), $post );
+		$statuses          = apply_filters( 'wp_parsely_trackable_statuses', array( 'publish' ), $post );
 		$cache[ $post_id ] = in_array( get_post_status( $post ), $statuses, true );
 		return $cache[ $post_id ];
 	}
 
+	/**
+	 * Check if the post's type is "publicly queryable."
+	 *
+	 * @since 2.6.0
+	 *
+	 * @param int|WP_Post $post Which post object or ID to check.
+	 * @return bool Is the provided post's type considered "public."
+	 */
+	public static function post_has_viewable_type( $post ) {
+		if ( function_exists( 'is_post_type_viewable' ) ) {
+			return is_post_type_viewable( $post->post_type );
+		}
+
+		/**
+		 * `is_post_type_viewable` was added in WordPress 4.4
```

Comment:
4.4 was first released in 2015. Are there any instances of the plugin running on older than that?

Reply:
I'm not sure about it, but it does make sense to retire that function at some point.

Reply:
https://github.com/Parsely/wp-parsely/issues/381